build_packages

```diff
@@ -222,9 +222,21 @@ break_dep_loop sys-apps/util-linux udev,systemd sys-apps/systemd cryptsetup
 break_dep_loop sys-apps/systemd cryptsetup
 
 info "Merging board packages now"
+setup_qemu_static "${BOARD_ROOT}"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" \
     @system coreos-devel/board-packages
 
+# Update bootengine.cpio and force rebuilt the initramfs:
+# as the ldconfig as called in the "/usr/sbin/update-bootengine"
+# script does not work for arm64 architecture, re-run it manually
+# and rebuild the kernel initramfs.
```

Comment:
Also, if it didn't work the first time wouldn't the build have already failed due to the kernel ebuild failing when it tries to use bootengine.cpio?